stdlib/tkinter/__init__.pyi

```diff
@@ -949,7 +949,7 @@ class Button(Widget):
         repeatinterval: int = ...,
         state: Literal["normal", "active", "disabled"] = ...,
         takefocus: _TakeFocusValue = ...,
-        text: str = ...,
+        text: Union[float, int, str] = ...,
```

Comment:
Type checkers automatically accept `int` too when you specify that something can be a `float`:

```suggestion
        text: Union[float, str] = ...,
```

Reply:
Made the changes.